tests/phpunit/tests/query/lazyLoadCommentMeta.php

```diff
@@ -26,6 +26,7 @@ public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
 	 * @ticket 57901
 	 *
 	 * @covers ::wp_queue_comments_for_comment_meta_lazyload
+	 * @expectedDeprecated wp_queue_comments_for_comment_meta_lazyload
```

Comment:
```suggestion
	 *
	 * @expectedDeprecated wp_queue_comments_for_comment_meta_lazyload
```

It's likely this will be added as a coding standard later so it's worth pre-empting.